bin/benchmark.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+java -cp "$(cd `dirname $0`/..; pwd)"'/tools/benchmark-cli/build/libs/benchmark-cli.jar:*' \
```

Comment:
It should be easy to add the [main class to the manifest](https://stackoverflow.com/questions/21721119/creating-runnable-jar-with-gradle) making the jar executable.  This can simplify the startup script and provide any easy bridge for Windows users to use this too without creating a .bat file. 

Unless we are intentionally trying to expose the classpath to the end user for extension, I advocate using executable jar's.